onnxruntime/core/providers/mkldnn/activation/activations.cc

```diff
@@ -0,0 +1,204 @@
+// Copyright(C) 2018 Intel Corporation
+// Licensed under the MIT License
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+
+#include "core/providers/mkldnn/mkldnn_common.h"
+#include "core/providers/mkldnn/activation/activations.h"
+#include "core/providers/mkldnn/mkldnn_fwd.h"
+
+namespace onnxruntime {
+namespace mkl_dnn {
+
+namespace {
+// Struct which encapsulates parameters for MKLDNN Pool primitive.
+struct ReluParams {
+  const mkldnn::memory::dims& src_dims;
+  const mkldnn::memory::dims& dst_dims;
+  size_t num_dimensions;
+
+  ReluParams(const mkldnn::memory::dims& src_dims, const mkldnn::memory::dims& dst_dims, 
+		size_t dimensions = 0)
```

Comment:
is dimensions intentionally a default parameter?

Reply:
actually for dimensions, can't we compute it from the src_dims? same comment for all these kernels...